### PR TITLE
Catalogue Review

### PR DESCRIPTION
**Pull request by elliottomlinson**
Changes to cards/illustrations/generator assets as discussed last night with @DuncanUszkay1 in our 'base set' deck review.

### CODE REVIEW

File: data/card_catalogue/movement.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
         "passive": null
       },
       "flavour": "Prison guard inside of a jail cell",
-      "image": "res/art/illustration/Platzfalt Step.png"
+      "image": "res/art/illustration/Swap.png"
```
Comment (elliottomlinson) [inline data/card_catalogue/movement.json:103]: closes #217

File: data/card_catalogue/anger.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
         "cast": "For one minute, NPCs will not become hostile unless they take damage",
         "hand": null
       },
-      "flavour": "There goes that thief with our jewlerly, what a great guy",
+      "flavour": "I haven't seen you in the vault before. First day on the job?.",
```
[the remark this answers is not in the document]
Comment (elliottomlinson) [reply data/card_catalogue/anger.json]: good catch, fixed in https://github.com/elliottomlinson/cardmaster/pull/214/commits/fdb78019246ca8730f8700e42bcbaf6ce4979feb

File: data/card_catalogue/movement.json [hunk 3]
```diff
@@ -99,8 +99,8 @@
         "active": "Switch positions with a visible NPC",
         "passive": null
       },
-      "flavour": "Prison guard inside of a jail cell",
-      "image": "res/art/illustration/Platzfalt Step.png"
+      "flavour": "Coach says you're up!",
```
Comment (DuncanUszkay1) [inline data/card_catalogue/movement.json:102]: This will be interesting with the train photo lol

File: data/card_catalogue/movement.json [hunk 4]
```diff
@@ -126,7 +126,7 @@
         "passive": null
```
Comment (DuncanUszkay1) [inline data/card_catalogue/movement.json:126]: Oh I think I figured out why these don't work, they still say active and passive

File: data/card_catalogue/npc/npc-generic.json [hunk 5]
```diff
@@ -1,30 +1,17 @@
...
-    "upgrade": "Empty the Clip",
+    "upgrade": null,
     "tier": "blue",
     "details": {
       "rules": {
         "cast": "Make an instant attack that cannot miss.",
         "hand": null
       },
-      "flavour": "A single, well timed strike can have greater impact than any barrage of blows",
+      "flavour": "From way downtown",
```
Comment (DuncanUszkay1) [inline data/card_catalogue/npc/npc-generic.json:11]: I feel like a `!` fits the vibe here

Comment (elliottomlinson) [reply data/card_catalogue/npc/npc-generic.json]: for sure, added in https://github.com/elliottomlinson/cardmaster/pull/214/commits/2458099cb4795aa89805ae6ccfc209597a30a1c0

File: data/card_catalogue/movement.json (hunk 4 above)
Comment (elliottomlinson) [reply data/card_catalogue/movement.json]: I'm embarrassed by how long I spent trying to figure out what was wrong with these lol, fixed in https://github.com/elliottomlinson/cardmaster/pull/214/commits/98ce1a99d278e575e166f9636aaeac8bc2a0c0ad

File: data/card_catalogue/unique-1.json [hunk 6]
```diff
@@ -112,7 +112,7 @@
         "cast": "Target willing character ceases to exist for up to 1 hour. They return to the last spot they existed in afterwards.",
         "hand": null
       },
-      "flavour": "Existence is rather tiring, it's advisable to take a break every once in a while",
+      "flavour": "Existence is tiring, why not take a break?",
```
Comment (DuncanUszkay1) [inline data/card_catalogue/unique-1.json:115]: Thoughts on just `Existence is tiring`?

File: data/card_catalogue/social.json [hunk 7]
```diff
@@ -99,7 +99,7 @@
         "cast": "Target visible NPC can no longer feel negative emotions towards target visible character",
         "hand": null
       },
-      "flavour": "There is no love without forgiveness, and no forgiveness without love",
+      "flavour": "Changed behaviour is the best apology",
```
[the remark this answers is not in the document]
Comment (elliottomlinson) [reply data/card_catalogue/social.json:102]: Down to change it if it doesn't feel obvious enough, idea was just as you suspected though to call out how it isn't an apology at all

File: data/card_catalogue/unique-1.json (hunk 6 above)
Comment (elliottomlinson) [reply data/card_catalogue/unique-1.json:115]: It feels a little obtuse to me by itself

File: data/card_catalogue/upgradeable-1.json [hunk 8]
```diff
@@ -227,7 +227,7 @@
     "details": {
       "rules": {
         "cast": "Something unlucky befalls target NPC",
-        "hand": null
+        "hand": "+1 to all d20 rolls"
```
[the remark this answers is not in the document]
Comment (elliottomlinson) [reply data/card_catalogue/upgradeable-1.json:230]: I'm cool with this one setting the precedent tbh

File: data/card_catalogue/unique-1.json [hunk 9]
```diff
@@ -343,7 +343,7 @@
     "tier": "green",
     "details": {
       "rules": {
-        "cast": "Choose an NPC. They are stuck in place for ten seconds.",
+        "cast": "Choose an NPC. Their speed is halved.",
```
[the remark this answers is not in the document]
Comment (elliottomlinson) [reply data/card_catalogue/unique-1.json]: I agree, didn't catch the loose end on that phrasing. Changed to 'until the end of the session' in https://github.com/elliottomlinson/cardmaster/pull/214/commits/228168f4efb743c1ba4ad55eb6eeb370ab00bf90

**Review by DuncanUszkay1 — APPROVED** [review]
Aside from the passive on `Hope(blue)` which should probably be for `Pray(green)`, I'm good to merge this.

For any continued flavour discussion it'll be best to look at the final card, so I'll save it for another set review

File: data/card_catalogue/upgradeable-1.json (hunk 8 above)
[the remark this answers is not in the document]
Comment (elliottomlinson) [reply data/card_catalogue/upgradeable-1.json:230]: Yeah we're very 'advantage' heavy instead, I think we agree that'd be a little too much for this card though but it's working everywhere else. Nice to have it on the table for the future I think.

File: data/card_catalogue/social.json (hunk 7 above)
[the remark this answers is not in the document]
Comment (DuncanUszkay1) [reply data/card_catalogue/social.json:102]: IMO let's leave it for now and update it while looking at the generated card in the next review session

**Comment by DuncanUszkay1** [comment]
Also the classic 63 commit PR, this man gits

File: data/card_catalogue/social.json [hunk 10]
```diff
@@ -125,7 +125,7 @@
         "cast": "Target NPC will believe your next statement",
         "hand": null
       },
-      "flavour": "A lie travels half way around the world before the truth puts on its shoes",
+      "flavour": "A bad excuse is worse than none",
```
[the remark this answers is not in the document]
Comment (elliottomlinson) [reply data/card_catalogue/social.json]: Gonna put it through just to get it in, we'll see it again in our next catalogue review so I'll mark this and https://github.com/elliottomlinson/cardmaster/pull/214#discussion_r729974824 resolved for now

**Pull request merged**